test/qunit/model/FHIRModel.unit.js

```diff
@@ -1172,4 +1173,12 @@ sap.ui.define([
 		assert.strictEqual(Object.keys(oOperationOutcome).length, 1, "Bundle Response without resource id is parsed without throwing an error");
 	});
 
+	QUnit.test("Submit Bundle with correct Etag", function(assert){
+		this.loadDataIntoModel("Patient2", this.sPatientPath2.substring(1));
+		this.oPropertyBinding3.setValue("2008-04-27");
+		this.oPropertyBinding4.setValue("female");
+		var mRequestHandles = this.oFhirModel1.submitChanges("patientDetails");
+		assert.equal("W/\"1\"", mRequestHandles.patientDetails.getBundle().getBundlyEntry(1).getRequest()["_sIfMatch"], "If-Match header is of correct syntax in bundle enteries ");
```

Comment:
To avoid accessing private members. You could use `mRequestHandles.patientDetails.getBundle().getBundlyEntry(1).getRequest().getBundleRequestData().ifMatch`

Reply:
done